src/main/java/org/opensearch/geospatial/ip2geo/common/GeoIpDataFacade.java

```diff
@@ -351,4 +344,13 @@ public static void putGeoData(
         client.admin().indices().prepareRefresh(indexName).execute().actionGet(timeout);
         client.admin().indices().prepareForceMerge(indexName).setMaxNumSegments(1).execute().actionGet(timeout);
     }
+
+    public AcknowledgedResponse deleteIndex(final String index) {
```

Comment:
Is this to delete any index? Should there be some restriction based on index name?

Reply:
Let me add a condition check on index name.